scripts/ci.sh

```diff
@@ -108,6 +110,13 @@ ${IOHK_OPS} "${GENERAL_OPTIONS[@]}" --config 'test-infra.yaml'  create deploy --
 banner 'Staging infra evaluated'
 fi
 
+if test -n "${WITH_BENCHMARK}"; then
+CLEANUP_DEPLS="${CLEANUP_DEPLS} test-bench"
+${IOHK_OPS}               new  --config 'test-bench.yaml'   --environment benchmark    ${COMMON_OPTIONS} 'test-stag'    ${CARDANO_COMPONENTS}
+${IOHK_OPS} ${GENERAL_OPTIONS} --config 'test-bench.yaml'   create deploy --dry-run
+banner 'Benchmark env evaluated'
```

Comment:
i think the topology file might be missing in this area, causing CI to fail

Reply:
Done.